dir.props

```diff
@@ -57,6 +57,7 @@
     <SourceBuiltPackagesPath>$(SourceBuiltBlobFeedDir)packages/</SourceBuiltPackagesPath>
     <SourceBuiltAssetsDir>$(SourceBuiltBlobFeedDir)assets/</SourceBuiltAssetsDir>
     <PrebuiltPackagesPath>$(ProjectDir)prebuilt/nuget-packages/</PrebuiltPackagesPath>
+    <PreviouslySourceBuiltPackagesPath>$(ProjectDir)prebuilt/source-built/</PreviouslySourceBuiltPackagesPath>
```

Comment:
There's a `_PreviouslySourceBuiltPackages` property in `dir.targets` that means something quite different--packages built already during the current build. Call this `PrebuiltSourceBuiltPackagesPath`? Or find a better name for `_PreviouslySourceBuiltPackages`?

I'm also wondering if we should settle on terms for the online build vs. tarball build and use those instead of relative terms.